src/utils/coins.ts

```diff
@@ -1,6 +1,7 @@
 const displayName = {
   usdt: 'USDT.e',
   usdc: 'USDC.e',
+  usdc_native: 'USDC',
```

Comment:
snake_case vs camelCase?

Reply:
Just following the pattern in xusd-contracts. Want me to change?